src/frontend/Ast_to_Mir.ml

```diff
@@ -106,7 +106,7 @@ and trans_idx = function
         raise_s
           [%message "Expecting int or array" (e.emeta.type_ : UnsizedType.t)] )
 
-and trans_exprs = List.map ~f:trans_expr
+and trans_exprs exprs = List.map ~f:trans_expr exprs
```

Comment:
Does this help something? I don't mind, just wondering if you thought this version was more clear.